xml/vk.xml

```diff
@@ -1851,6 +1851,9 @@ server.
             <member optional="true"><type>VkDeviceSize</type>                             <name>sequencesCountOffset</name></member>
             <member optional="true"><type>VkBuffer</type>                                 <name>sequencesIndexBuffer</name></member>
             <member optional="true"><type>VkDeviceSize</type>                             <name>sequencesIndexOffset</name></member>
+            <implicitexternsyncparams>
+                <param>the sname:VkCommandPool that pname:targetCommandBuffer was allocated from</param>
+            </implicitexternsyncparams>
```

Comment:
Not sure this makes sense either. The `targetCommandBuffer` is supposed to already have a space reserved.